app/controllers/api/auth/podcast_imports_controller.rb

```diff
@@ -9,6 +9,12 @@ class Api::Auth::PodcastImportsController < Api::BaseController
 
   filter_params :url
 
+  def create_resource
+    super.tap do |podcast_import|
+      podcast_import.user_id ||= current_user.id
```

Comment:
There's a user_id on podcast imports?  I thought it was just account and series.